driver/lstm_verify_gemm.hpp

```diff
@@ -141,8 +141,8 @@ void RunLSTMForwardGEMMCPUVerify(miopenHandle_t handle,
     // forward emulator
     for(int li = 0; li < numlayer; li++)
     {
-        int hid_shift = li * batch_n * hy_stride;
-        int hx_shift  = li * in_n.at(0) * h_stride;
+        size_t hid_shift = li * batch_n * hy_stride;
```

Comment:
[Informative] @shurale-nkn Just to remind: if batch_n were not of the size_t type, then changing the type of hid_shift from int to size_t won't prevent overflow (in this line), because li and hy_stride are of type int. And now, as the result of `(li * batch_n * hy_stride)` is of type size_t, so you can use `auto` in the definition of hid_shift.

Reply:
> [Informative] @shurale-nkn Just to remind: if batch_n were not of the size_t type, then changing the type of hid_shift from int to size_t won't prevent overflow (in this line), because li and hy_stride are of type int. And now, as the result of `(li * batch_n * hy_stride)` is of type size_t, so you can use `auto` in the definition of hid_shift.

@atamazov as long as we using rocblas with offset in int we can and will overflow, This is not an attempt to fix something. it's just that I've made changes locally for some variables. Globally, I know that this code does not support large values, but changing this will require not only correct type conversion.

Reply:
@shurale-nkn In other words, you've changed `int` to `size_t` in many places due to some reason that is not related to fixing potential or real overflows. Okay.
[resolved]